interface/modules/custom_modules/oe-module-claimrev-connect/public/index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+    $tab="home";
+?>
```

Comment:
need a require_once for globals.php to ensure only authenticated users can get to this script

Reply:
done